app/src/Vues/LoginVue.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import AuthModel from "../scripts/Models/AuthModel";
+
+function Login() {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [error, setError] = useState("");
+
+    const handleSubmit = async (e: React.FormEvent) => {
+        e.preventDefault();
+        setError("");
+        
+        try {
+            const auth = new AuthModel(username, password);
+            const response = await auth.login();
+            
+            if (response.isError()) {
+                setError("Identifiants incorrects");
```

Comment:
You should make sure that the error really corresponds to a credentials issues, and not another error from the server.